.travis/push-to-nexus.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+openssl aes-256-cbc -K $encrypted_57e4d0a26fa7_key -iv $encrypted_57e4d0a26fa7_iv -in signing.gpg.enc -out signing.gpg -d
+gpg --import signing.gpg
+
+export GPG_EXECUTABLE=gpg
```

Comment:
This is slightly tangential, but `GPG_EXECUTABLE=gpg mvn -DskipTests ...` would make the scope of the env var much clearer. It's not such a big deal _here_, but in the `build.sh` the number of env vars is growing and their intended semantics are becoming less well-defined.